build-support/docker/Dev.dockerfile

```diff
@@ -1,2 +1,8 @@
 FROM hashicorp/consul-k8s:latest
+
+# change the user to root so we can install stuff
+USER root
+RUN apk update && apk add iptables
+USER consul-k8s
```

Comment:
we should be able to remove this after we release, but for now it's useful to have for the dev images.